wfe/context.go

```diff
@@ -65,6 +65,11 @@ func (th *topHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		logEvent.Endpoint = r.URL.String()
 	}
 	defer th.logEvent(logEvent)
+	defer func() {
+		if logEvent.Requester > 0 {
+			w.Header().Set("Boulder-Requester", fmt.Sprintf("%d", logEvent.Requester))
```

Comment:
Since refactoring `verifyPOST` is not something I would be fully comfortable with at this point, I chose the former. It leads to a bit of duplication, but after each call to `verifyPOST` the Boulder-Requester header is added. Avoiding passing `response` into verifyPOST.